recipes/sassc/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SasscConan(ConanFile):
+    name = "sassc"
+    license = "MIT"
+    homepage = "libsass.org"
```

Comment:
The homepage now is `https://sass-lang.com/libsass`. The old one is redirecting. Debian points to the sass-lang too.

Reply:
Let's see what other have to say about this PR. I'll  modify this if there is another change to be made in order to not waste CI time and existing review/

Reply:
You can move the url, actually, it would be nice if you add https://